### ISSUE TITLE
bug in handling quote marks

### ISSUE THREAD

**Issue by sharoncorrell** [silnrsi/pysilfont#65]
My FEAX looks like:
```
lookup ReorderInsert {
      sub uni1C00' uni1C34 by uni1C34 uni1C00;
} ReorderInsert;
```

The resulting FEA is:
```
lookup ReorderInsert {
    sub uni1C00 uni1C34 by uni1C34' uni1C00';
} ReorderInsert;
```

The quote marks on the FEA are wrong.

**Comment by mhosken** [comment]
The result should be:
```
lookup ReorderInsert {
    sub uni1C00' uni1C34 by uni1C34 uni1C00;
} ReorderInsert;
```
Yes. There is a bug here.

I would suggest reading [ligatureformation](https://github.com/OpenType/opentype-layout/blob/master/docs/ligatures.md) whenever considering reordering.

**Comment by mhosken** [comment]
Perhaps fixed in dd8a519ceebfd0c72f462e3b3ff5dc78afc77dec

**Comment by devosb** [comment]
[Test added](https://github.com/silnrsi/feax/commit/286c37be3b71eda6e258364f070bfa4cb4111cfe#diff-c8f47b733f54610ea93d9cdf60b88713497dd520b64033a69a75610700ac17b8) to the new feaxlib module in https://github.com/silnrsi/feax/commit/286c37be3b71eda6e258364f070bfa4cb4111cfe
